test/c3/appframework/appframework_test.go

```diff
@@ -2500,4 +2501,132 @@ var _ = Describe("c3appfw test", func() {
 
 		})
 	})
+
+	// CSPL-1688
+	XContext("Single Site Indexer Cluster with Search Head Cluster (C3) and App Framework", func() {
+		It("integration, c3, appframeworkc3, appframework: can deploy a C3 SVA with App Framework enabled, disable it via the conf file and by removing from s3 bucket", func() {
```

Comment:
Nit: This would need some clarification imho, for ex something like "can deploy a C3 SVA with App Framework enabled, install an app, then disable it and eventually remove it from app source"